package.json

```diff
@@ -30,7 +30,7 @@
     "eslint": "^6.8.0"
   },
   "scripts": {
-    "build": "./src/trackers/process-crawl.js && ./src/trackers/build-trackers.js",
+    "build": "NODE_OPTIONS=--max-old-space-size=8096 ./src/trackers/process-crawl.js && ./src/trackers/build-trackers.js",
```

Comment:
TIL! `NODE_OPTIONS` magically passes everything to node? (we should probably prefix all TR scripts with that 😆)